src/console_node/httpUtils.go

```diff
@@ -66,7 +66,7 @@ func SendResponseJSON(w http.ResponseWriter, sc int, data interface{}) {
 // Helper function to execute an http POST command
 func postURL(URL string, requestBody []byte, requestHeaders map[string]string) ([]byte, int, error) {
 	var err error = nil
-	log.Printf("postURL URL: %s\n", URL)
+	//log.Printf("postURL URL: %s\n", URL)
```

Comment:
Same question -- should we just remove this? Or put it behind some conditional, if you're keeping it here for future debugging?

Reply:
Ibid.